test/e2e/common/node/pod_resize.go

```diff
@@ -1409,8 +1420,8 @@ func doPodResizeErrorTests() {
 	for idx := range tests {
 		tc := tests[idx]
 		ginkgo.It(tc.name, func(ctx context.Context) {
-			ginkgo.By("waiting for the node to be ready", func() {
-				if !supportsInPlacePodVerticalScaling(ctx, f) || framework.NodeOSDistroIs("windows") || isRunningOnArm64() {
+			ginkgo.By("check if in place pod vertical scaling is supported", func() {
```

Comment:
Can this check be moved to the top level, just inside the SIGDescribe at the bottom of this file?

Reply:
Does a `ginkgo.BeforeAll(...)` work there?

Reply:
Works, thanks.